experiment-setup/execution/data_collection.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
```

Comment:
I assume this stuff is from Helge, no? I have no idea to set up the otii arc.

Reply:
No, it is not related with otii. It is just an lib to build the path e.g. where to save the csv files. Looks easier using it.

Reply:
He is just getting the extra argument that is being passed when running the script and setting as a path to save the stuff. Take a look in the main execution file.
`Path(sys.argv[1])`